jmix-flowui/flowui-kit/flowui-kit.gradle

```diff
@@ -26,4 +26,4 @@ dependencies {
     api("com.vaadin:vaadin-core:$vaadinFlowVersion") {
```

Comment:
Why do we need an explicit version here? Won't it be taken from Jmix BOM?

Maybe just `api("com.vaadin:vaadin-core")` can work and that we can remove vaadinFlowVersion from settings gradle?